implementation/src/main/java/io/smallrye/mutiny/infrastructure/Infrastructure.java

```diff
@@ -422,4 +403,71 @@ public interface OperatorLogger {
          */
         void log(String identifier, String event, Object value, Throwable failure);
     }
+
+    private static class MutinyInterceptorComparator implements Comparator<MutinyInterceptor> {
+
+        private static final MutinyInterceptorComparator INSTANCE = new MutinyInterceptorComparator();
+
+        private MutinyInterceptorComparator() {
+        }
+
+        @Override
+        public int compare(MutinyInterceptor o1, MutinyInterceptor o2) {
+            return Integer.compare(o1.ordinal(), o2.ordinal());
+        }
+    }
+
+    private static class AlwaysTrueBooleanSupplier implements BooleanSupplier {
+
+        private static final AlwaysTrueBooleanSupplier INSTANCE = new AlwaysTrueBooleanSupplier();
+
+        private AlwaysTrueBooleanSupplier() {
```

Comment:
We don't need a constructor here since the class is `private` 😉

Reply:
Good point